ranger/core/main.py

```diff
@@ -119,6 +119,9 @@ def main(
         FileManagerAware.fm_set(fm)
         load_settings(fm, args.clean)
 
+        if args.choosedir:
+            SettingsAware.settings.global_inode_type_filter = 'd'
```

Comment:
Where should these constants be declared/stored? The value being assigned will be the same, but I can understand wanted a constant for readability and maintenance purposes.